### ISSUE TITLE
New version notice disappears very fast

### ISSUE THREAD

**Issue by sorawee** [brownplt/code.pyret.org#245]
The newer version of CPO notice appears in like half a second and fades away very quickly. Can we make it permanent with a close button so that users can click close when they want to, but will definitely see it?

**Comment by jpolitz** [comment]
Yeah that's a good idea. Just need to make a third widget – we have `stickMessage` and `flashMessage`, we need `dismissableMessage`.

**Comment by jpolitz** [comment]
Note that it will show up every 2-3 minutes to nag them, and then fade again.

**Comment by schanzer** [comment]
@jpolitz do we still make use of the version parameter?

**Comment by schanzer** [comment]
@jpolitz still worth keeping open?
